app/src/main/res/values/strings.xml

```diff
@@ -851,6 +851,15 @@
     <string-array name="select_light_sensor">
         <item>Built In</item>
         <item>BH1750</item>
+        <item>TSL2561</item>
+    </string-array>
+    <string-array name="select_lux_gain">
+        <item>No gain</item>
+        <item>1x</item>
+        <item>16x</item>
+        <item>500mLx</item>
```

Comment:
Why are these having different units? If it is a gain, shouldn't it be unit-less?

Reply:
The previous implementation for BH1750 sensor in Sensors instrument was like this. I'll change them.